src/langsmith/deployments.mdx

```diff
@@ -19,9 +19,9 @@ A typical deployment workflow consists of the following steps:
 
 <Info>
 **Prerequisites:** Before deploying applications, you need a LangSmith instance to deploy to. Choose a [hosting option](/langsmith/hosting) first:
-- [**Cloud**](/langsmith/cloud): Fully managed
-- [**Hybrid**](/langsmith/hybrid): Enterprise option for data residency requirements
-- [**Self-hosted**](/langsmith/self-hosted): Full control and data isolation
+- [**Cloud**](/langsmith/cloud): Fully managed with automated CI/CD from GitHub.
```

Comment:
For documentation maintainability, I would replace `GitHub` with `git repository` (generic). In the future, we may add integrations with GitLab, BitBucket, etc.